### PR TITLE
Patch ibc testing for test msg ibc request

### PR DESCRIPTION
**Pull request by Benzbeeb**
Fixed: #XXXX

## Please ensure the following requirements are met before submitting a pull request:

- [ ] The pull request is targeted against the correct target branch
- [ ] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [ ] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [ ] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [ ] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -599,6 +599,32 @@ func (app *BandApp) GetSubspace(moduleName string) paramstypes.Subspace {
...
+// GetScopedIBCKeeper implements the TestingApp interface.
+func (app *BandApp) GetScopedIBCKeeper() capabilitykeeper.ScopedKeeper {
+	return app.ScopedIBCKeeper
+}
+
+// GetTxConfig implements the TestingApp interface.
+func (app *BandApp) GetTxConfig() client.TxConfig {
+	return MakeEncodingConfig().TxConfig
+}
+
```
Comment (taobun) [inline app/app.go:627]: Move to testapp

**Review by taobun — CHANGES_REQUESTED** [review]

File: x/oracle/ibc_test.go [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+		path.EndpointA.ChannelID,
+		clienttypes.ZeroHeight(),
+		1577924000000000000,
+	)
+	expectCommitment := channeltypes.CommitPacket(suite.chainB.Codec, responsePacket)
+	commitment := suite.chainB.App.IBCKeeper.ChannelKeeper.GetPacketCommitment(suite.chainB.GetContext(), path.EndpointB.ChannelConfig.PortID, path.EndpointB.ChannelID, 1)
+	suite.Equal(expectCommitment, commitment)
+}
+
+func TestTransferTestSuite(t *testing.T) {
```
Comment (taobun) [inline x/oracle/ibc_test.go:128]: Transfer?

**Review by taobun — CHANGES_REQUESTED** [review]

**Review by taobun — APPROVED** [review]
LGTM thanks!

**Comment by songwongtp** [comment]
merge https://github.com/bandprotocol/chain/pull/89 into this first

**Review by songwongtp — APPROVED** [review]

**Pull request merged**